backend/app/database/alembic/versions/2024_07_22_1630-2e48ad16e9f7_add_platforms_settings_table.py

```diff
@@ -22,6 +22,9 @@
 
 
 def upgrade() -> None:
+    class PlatformTypes(StrEnum):
```

Comment:
Shouldn't this be part of the source code instead of an Alembic migration?

Reply:
edit: it is probably there to be able to remove the old enum? weird syntax but fine for now

Reply:
Yes, this is required to remove redundant enums.py